pkg/apis/kubeflow.org/v1/common_types.go

```diff
@@ -35,6 +35,12 @@ const (
 
 	// JobRoleLabel represents the label key for the job role, e.g. master.
 	JobRoleLabel = "training.kubeflow.org/job-role"
+
+	// KubeflowJobsController represents the value of the default jobs controller
+	KubeflowJobsController = "kubeflow.org/training-operator"
+
+	// MultiKueueController represents the value of the only allowed external controller except for KubeflowJobsController
```

Comment:
```suggestion
	// MultiKueueController represents the value of the only allowed external controller except for KubeflowJobsController
```
```suggestion
	// MultiKueueController represents the MultiKueue controller
```
nit: I think it is unnecessarily repetitive to say here that MultiKueue is the only supported.